core/scss/components/brand-bar/_brand-bar.scss

```diff
@@ -0,0 +1,32 @@
+@charset "UTF-8";
+
+//
+// Brand Bar
+//
+// Stanford brand bar with wordmark logo. This sits at the very top of a website.
+//
+// .su-brand-bar--default - Default brand bar with white wordmark on cardinal red background
+// .su-brand-bar--bright - White wordmark on bright red background
+// .su-brand-bar--dark - White wordmark on dark grey background
+// .su-brand-bar--white - cardinal red wordmark on white background
+//
+// Markup: ../../templates/components/brand-bar/brand-bar.twig
+//
+// Style guide: Components.Brand Bar
+//
+
+#su-brand-bar,
+.su-brand-bar--default {
+
+  @include centered-grid-container;
+  width: 100%;
+  height: 30px;
+  background-color: $color-cardinal-red;
+
+  .su-brand-bar__logo {
```

Comment:
This can be popped out of the parent wrapper and be on it's own if it is the default item. This makes it easier to override in variants.

Reply:
Sure! It's been taken out of parent wrapper in default scss file.